web/pdf_viewer.js

```diff
@@ -757,7 +757,8 @@ var PDFViewer = (function pdfViewer() {
       return new AnnotationLayerBuilder({
         pageDiv: pageDiv,
         pdfPage: pdfPage,
-        linkService: this.linkService
+        linkService: this.linkService,
+        downloadManager: new DownloadManager()
```

Comment:
I'm still wondering if it wouldn't make more sense to pass in to `PDFViewer`, similar to this: https://github.com/mozilla/pdf.js/blob/master/web/pdf_viewer.js#L42. And the do (in `PDFViewer`), `this.downloadManager = options.downloadManager || null;`

The reason for asking is that `DownloadManager` won't be available in the `pageviewer` component (the `LinkService` is stubbed out too, so I wouldn't worry about that).
Similarily, you probably want to check that it's defined before calling it here: https://github.com/mozilla/pdf.js/pull/6988/files#diff-1097e3b131ba22f8e8a6c2f20e3ab5ecR786.